prod/westeurope/internal/api/storage_test-users/account/terragrunt.hcl

```diff
@@ -0,0 +1,24 @@
+# Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_storage_account?ref=v3.0.3"
+}
+
+inputs = {
+  name                     = "testusers"
```

Comment:
If this storage is meant for what we call `beta` users, why not call it accordingly?

Reply:
ok

Reply:
Done in [810e84b](https://github.com/pagopa/io-infrastructure-live-new/pull/498/commits/810e84b3f93113aa16a36fd2b2fb75086bbab911)